src/cbmc_starter_kit/template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -216,6 +226,10 @@ def run_build(litani, jobs, fail_on_proof_failure):
         logging.critical("Failed to run litani run-build")
         sys.exit(1)
 
+    if summarize:
```

Comment:
I cry a little bit whenever I see something that Go's `defer` keyword effortlessly solves...